doc/architecture/20170816-audit-plan-to-model-audit-data.md

```diff
@@ -8,7 +8,7 @@ An **inventory plan** is a set of rules that define the content that belongs to
 
 The **inventory sheet** refines an inventory plan by excluding formats by adding/removing sets of content items.
 
-**Audit Sheet**: is what a the department is going to audit. It is the inventory plan refined twice.
+**Audit Sheet**: is what a the department is going to audit. It is the result of multiple refinements (`theme` -> `inventory plan` -> `inventory sheet` -> `audit sheet`), and contains the list of content items that need auditing.
```

Comment:
Are we still using themes?

Reply:
We are not using themes as of now. But they still exist, so it is good to define them here as they have been a source of misunderstandings in the last weeks.